docs/source/federation/migrating-from-stitching.md

```diff
@@ -215,6 +184,68 @@ extend type Reservation @key(fields: "id") {
 
 The `Reservation.user` resolver will receive an `id` (of the reservation) and a `userId`. You can use the `userId` or anything else defined on the `Reservation` to lookup our user. For this example, the `userId` of a reservation is all you would need.
 
-### Starting up a new Gateway
+### Modifying the schema
+
+Once you have fully migrated your graph to federation, it should look identical to your stitching gateway's schema, unless you decided to remove unused types along the way. At this point, you can fully migrate all traffic and begin to modify your existing schema to take advantage of federation. For more information on how to do this, check out [this guide](/federation/core-concepts/).
+
+## Comparison with schema stitching
```

Comment:
Should this section be a part of the introduction?  This context is pretty helpful and provides a "roadmap" for how to think about migration as the document continues.

Reply:
We actually just removed it and pointed to the blog post instead 👀 I think the doc is useful to have elsewhere, but it doesn't exactly make sense within the "guide" in its current state.